### PR TITLE
[8.x] Added contains Method to Arr Helper

### PR DESCRIPTION
**Pull request by hmennen90**
This PR adds a contains Method to Arr helper which simply Checks if a given Value is Inside a Array - supporting Multi Dimensional Arrays.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -155,7 +155,10 @@
         "wildbit/swiftmailer-postmark": "Required to use Postmark mail driver (^3.0)."
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "allow-plugins": {
+            "composer/package-versions-deprecated": true
+        }
```
Comment (driesvints) [inline composer.json:161]: Can you revert this? It's already added.

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions! 

If you feel absolutely certain that this code corrects a bug in the framework, please "**@**" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

**Comment by Wiruu** [comment]
@taylorotwell Hello, I was just about to write an issue about this, its not a bug but an enhancement since the Arr helper methods of has and hasAny check for array keys and not values, trying to write as less code as possible and not manipulate original array, php function in_array() serves well... I believe this would be a great addition to the Arr helper...

**Comment by aurawindsurfing** [comment]
I second this request. It feel unnatural to Laravel ecosystem that this helper is missing.

**Pull request closed without merging**